test/mockS3UploadStream.js

```diff
@@ -0,0 +1,27 @@
+const Writable = require('stream').Writable;
+
```

Comment:
It would be good to include a comment here indicating that this is a mock version of S3UploadStream from `s3-uploda-stream`.  The implementation looks good.

Reply:
Great! Will do all the requested changes, thank you!